docs/custom_loading.rst

```diff
@@ -161,16 +161,16 @@ This again will be done in a separate python file and placed in the user's
 
     @custom_writer("fits-writer")
     def generic_fits(spectrum, file_name, **kwargs):
-        flux = spectrum.flux.value
-        disp = spectrum.spectral_axis.value
+        flux = spectrum.flux
+        disp = spectrum.spectral_axis
         meta = spectrum.meta
 
         tab = Table([disp, flux], names=("spectral_axis", "flux"), meta=meta)
 
         tab.write(file_name, format="fits")
 
 Please note that it is assumed that the axis (of the ``flux`` and ``spectral_axis``)
-units information is stored in the ``meta`` parameter as introduced in the definition
+units information is stored in ``Table`` object as introduced in the definition
```

Comment:
Sorry, it was not clear to me what you meant by "introduced in the definition" here; anyway this is not assuming anything, since these two `Spectrum1D` attributes _must_ have `unit` properties, and a `Table` will preserve them as `Quantity` columns.
Also added my suggestions for consistent naming above (see my comment in the main discussion); I had not noticed that this section was using yet different names (and also duplicating the function definition of `generic_fits` if followed verbatim).

Reply:
Sure, have incorporated your suggestion for consistent naming.